### PR TITLE
feat: Skills & Commands (Layer 2)

### LINKED ISSUE #1: Skills & Commands (Layer 2)
Rewrite all agent skills and commands for GitHub Issues + hierarchical branching. This is the core value of the project.

## What
- AGENTS.md: workflow overview, GH Issues conventions, /plan and /work usage, issue structure requirements
- Planner skill: explore codebase, discuss, file GH Issues as sub-issues with dependencies via gh CLI
- Coordinator/work skill: recursive /work #N — create/reuse branch, rebase, implement (leaf) or orchestrate (non-leaf), handle review findings on re-runs
- Implementer skill: pure dev (tests first, implement, quality gates) — remove beads references
- 3 reviewer skills: create GH child issues with blocking/should-fix/suggestion labels
- Plan reviewer skill: validate filed issues against codebase
- /plan and /work commands
- .claude/settings.json (gh and git permissions)

## Why
The skills and commands are what users interact with daily. Everything else (Actions, installer) builds on top of these.

## Reference
See docs/design.md, Layer 2 section.

### PR DESCRIPTION
**Pull request by jdelfino**
Add all skills and commands for the GitHub Issues workflow.

Skills:
- planner: Collaborative epic planning with GitHub Issues
- coordinator: Worktree setup, branch/PR management, delegates to implementer
- implementer: Test-first development (spawned by coordinator)
- reviewer-correctness: Bugs, security, error handling
- reviewer-tests: Test quality and coverage
- reviewer-architecture: Duplication, pattern consistency
- reviewer-plan: Validate filed issues before implementation

Commands:
- /plan: Decompose work into GitHub Issues with sub-issues
- /work: Make progress on an issue (implement, fix, continue)

Also adds:
- AGENTS.md: Workflow conventions, GitHub API reference
- CLAUDE.md: Starter template for target projects
- settings.json: Permissions for gh and git CLI

Key design decisions:
- Always use worktrees (multiple epics may be in progress)
- Coordinator always delegates to implementer subagents
- PR accumulates Fixes #N as tasks complete
- All issues close atomically on merge

Fixes #1, #7, #8, #9, #10, #11, #12, #13, #14

### CODE REVIEW

File: workflow/.claude/skills/coordinator/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+name: coordinator
+description: Single entry point for /work #N. Creates branch off main for parent issues, implements leaf issues on parent's branch, orchestrates children. Invoked via /work.
+user_invocable: true
+---
+
+# Coordinator
+
+You are the work coordinator. `/work #N` is the single entry point for all implementation. You either implement the work directly (for leaf issues) or orchestrate children (for parent issues).
+
+**Two-level model:**
```
Comment (jdelfino) [inline workflow/.claude/skills/coordinator/SKILL.md]: Is this relevant or informative here?

File: workflow/.claude/skills/coordinator/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+git pull --rebase origin main
+```
+
+If the leaf has no parent, it's a standalone task — create a branch off main.
+
+---
+
+## Phase 3: Implementation
+
+### For Leaf Issues — Implement Directly
```
Comment (jdelfino) [inline workflow/.claude/skills/coordinator/SKILL.md]: I still think this should not be treated as a special case. An issue with no children can be treated as an epic with one child (itself), and handled the same way (subagent implementer)

File: workflow/.claude/skills/coordinator/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+There is no separate `/cleanup` command. `/work` handles both initial implementation and fixing review findings.
+
+---
+
+## Quality Gates
+
+Before pushing any work:
+
+```bash
```
Comment (jdelfino) [inline workflow/.claude/skills/coordinator/SKILL.md]: These are project-specific, remove. "All relevant build, test, and lint steps must pass" is probably the best we can do at this generic level

File: workflow/.claude/skills/implementer/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+# Implementer
+
+Follow these phases **in strict order**. Do not skip phases. Do not proceed until the current phase's gate is satisfied.
+
+This skill covers development only — no issue tracking, no branch management, no commits. The coordinator handles those.
+
+## Principles
+
+- Never silently work around problems. Throw errors for missing env vars, invalid state, missing dependencies.
```
Comment (jdelfino) [inline workflow/.claude/skills/implementer/SKILL.md]: "Fail fast and loudly if something is wrong"

File: workflow/.claude/skills/implementer/SKILL.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+- **Are there gaps?** Specifically:
+  - Happy path for new/changed behavior
+  - Error paths and edge cases
+  - Regression test if this is a bug fix (a test that would have caught the original bug)
+  - Boundary conditions
+
+### Step 3: Evaluate integration test needs
+
+Integration tests are needed when changes affect:
+- Repository/persistence layer (database queries, data mapping)
```
Comment (jdelfino) [inline workflow/.claude/skills/implementer/SKILL.md]: These can be listed as examples, but are project-specific

File: workflow/AGENTS.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
+# AGENTS.md
```
Comment (jdelfino) [inline workflow/AGENTS.md]: Much of this file seems like it belongs in individual skills. Re-evaluate this. Only keep information that is truly common among all skills and should be in context for every interaction.
